meta-mbl-distro/recipes-core/mbl-initramfs-init/files/initramfs-init-script.sh

```diff
@@ -65,38 +74,51 @@ mbl-watchdog-init --timeout ${MBL_WATCHDOG_TIMEOUT_SECS} --device ${MBL_WATCHDOG
 set -e
 echo "Booting from init script in initramfs" || true
 
+# We'll mount the factory config partition first because that contains
+# information about the partition numbers of the other partitions. That will
+# allow us to mount other partitions without relying on their labels. This is
+# important in the case of the rootfs because we won't be able to distinguish
+# between the two banks of rootfs based on their labels.
+
 # Workaround findfs failure on Raspberry Pi 3: unable to resolve 'LABEL=xxxx'
 # We need to wait until the kernel mmc driver is up and all storage partitions
 # populated.
-while ! findfs LABEL=${LOG_PARTITION_LABEL}>/dev/null 2>&1 ; do
+while ! findfs LABEL=${FACTORY_CONFIG_LABEL}>/dev/null 2>&1 ; do
     sleep 0.01
 done
 
+FACTORY_CONFIG_DEVICE_PATH="$(findfs "LABEL=${FACTORY_CONFIG_LABEL}")"
+mount_or_fsck "$FACTORY_CONFIG_DEVICE_PATH" "$FACTORY_CONFIG_MOUNT_POINT" -o "$FACTORY_CONFIG_MOUNT_OPTS"
+
+
 # The boot flags (which indicate which rootfs bank to use) are temporarily
 # stored in the log partition. Mount the log partition so we can check them.
 
-LOG_PARTITION="$(findfs "LABEL=${LOG_PARTITION_LABEL}")"
-mount_or_fsck "$LOG_PARTITION" "$LOG_MOUNT_POINT" -o "$LOG_MOUNT_OPTS"
+FLASH_DEVICE_PATH="${FACTORY_CONFIG_DEVICE_PATH%p[0-9]*}"
```

Comment:
```suggestion
STORAGE_DEVICE_PATH="${FACTORY_CONFIG_DEVICE_PATH%p[0-9]*}"
```

Reply:
I agree, `STORAGE` is a better word. I'll do a search and replace in my text editor rather than taking the suggestions via GitHub though so that I can convince myself I've found all of the places that need changing...

Reply:
https://github.com/ARMmbed/meta-mbl/pull/795/commits/a87553e4544b44375501d034f971ed66c2d1af48